Sources/Sentry/Public/SentryHub.h

```diff
@@ -56,7 +56,21 @@ SENTRY_NO_INIT
     NS_SWIFT_NAME(startTransaction(name:operation:));
 
 /**
- * Creates a transaction bound to the hub and returns the instance.
+ * Creates a transaction, bound it to the hub and returns the instance.
+ *
+ * @param name The transaction name.
+ * @param operation Short code identifying the type of operation the span is measuring.
+ * @param bindToScope Indicates whether the new transaction should be bind to the scope.
+ *
+ * @return The created transaction.
+ */
+- (id<SentrySpan>)startTransactionWithName:(NSString *)name
```

Comment:
`bindToScope` should be false by default if one calls `startTransaction` without `bindToScope`